presto-main/src/main/java/io/prestosql/sql/rewrite/DescribeInputRewrite.java

```diff
@@ -113,10 +114,10 @@ protected Node visitDescribeInput(DescribeInput node, Void context)
 
             // return the positions and types of all parameters
             Row[] rows = parameters.stream().map(parameter -> createDescribeInputRow(parameter, analysis)).toArray(Row[]::new);
-            Optional<String> limit = Optional.empty();
+            Optional<String> limitRowCount = Optional.of("ALL");
```

Comment:
Instead of generating a `LIMIT ALL`,  use an `Optional<Node> limit = Optional.empty()` and assign `Optional.of(new Limit("0"))` in the `if ()` block below.